algorithms/Python/graphs/python-dijsktra.py

```diff
@@ -0,0 +1,110 @@
+# A Dijkstra's algorithm implementation in Python
```

Comment:
Please remove this file that is with incorrect name.

Reply:
Hi,
File already renamed 🙂